lib/WebService/Async/Onfido.pm

```diff
@@ -1034,6 +1034,52 @@ sub countries_list {
     });
 }
 
+=head2 is_country_supported
+
+Returns 1 if country supported and 0 for unsupported
+
+=cut
+
+sub is_country_supported {
+    my ($self, $country_code) = @_;
+
+    my $country_list = countries_list($self)->get;
+
+    return $country_list->{$country_code};
+}
+
+=head2 supported_documents_list
+
+Returns an array of hashes of supported_documents for each country
+
+=cut
+
+sub supported_documents_list {
+    my $path = Path::Tiny::path(__DIR__)->parent(3)->child('share/supported_documents.json');
+    $path = Path::Tiny::path(
+        File::ShareDir::dist_file(
+            'WebService-Async-Onfido',
+            'supported_documents.json'
+        )
+    ) unless $path->exists;
+    my $supp_docs = decode_json_text($path->slurp_utf8);
```

Comment:
better to use full name `$supported_documents`

Reply:
changed